tests/utils.py

```diff
@@ -32,3 +33,17 @@ def check_obj(
         obj_values = omit(obj, skip_keys)
     values = omit(values, skip_keys)
     assert_equals(obj_values, values)
+
+
+def is_os_windows_and_py_lt_3_9() -> bool:
+    return (
+        os.environ.get("GITHUB_MATRIX_OS") == "windows-latest"
+        and os.environ.get("GITHUB_MATRIX_PYTHON", "2") < "3.9"
```

Comment:
Why `"2"`?

Reply:
So: can't leave it empty or use None, because python raises an issue (can't compare None to a string - I was surprised that python was so smart to check that out!).

I set "2" to make the condition true in case the python version cannot be detected (if I am not sure, better skip it).

I was not totally happy about it, I would be glad to hear a suggestion.

Reply:
Hm, I see. Nothing comes to mind except for this I guess
```suggestion
        and os.environ.get("GITHUB_MATRIX_PYTHON", "") < "3.9"
```

Reply:
change done